templates/site/index.html

```diff
@@ -22,80 +22,63 @@ <h1 class="text-center">{{ page_heading }}</h1>
 </div>
 {% endblock %}
 
+
+
+{% set table_data = [
+                {"name": "Steve Easterbrook", "institution": "University of Toronto", "department": "Computer Science"},
+                {"name": "Samuel Foucher", "institution": "Centre de recherche informatique de Montréal (CRIM)", "department": "Vision and Imaging Team"},
+                {"name": "Nathan Gillett", "institution": "Climate Research Division", "department": "Environment and Climate Change Canada"},
```

Comment:
as discussed: swap institution and department

Reply:
Swapped institution and department for Nathan Gillett.